src/main/webapp/history/history.js

```diff
@@ -19,51 +18,337 @@
+			chartPlaceHolder.echart = chart;
+			
+			const textColor = getComputedStyle(document.body).getPropertyValue('--darkreader-text--text-color') || getComputedStyle(document.body).getPropertyValue('--text-color') || '#222';
+			const showSettings = document.getElementById(settingsDialogId);
+			let darkMode = getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background')
+			darkMode = darkMode !== undefined && darkMode !== null && darkMode !== ''
+			console.log('darkMode: ' + darkMode)
+			let series = model.duration.series.concat(model.result.series)
+			series.forEach(s => s.emphasis = {
+				disabled: true
+			});
+			const options = {
+				animation: false,
+				darkMode: darkMode,
+				//backgroundColor: getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background') || getComputedStyle(document.body).getPropertyValue('--bs-body-bg') || '#fff',
+				toolbox: {
+					feature: {
+					  dataZoom: {
+						yAxisIndex: 'none'
+					  },
+					  restore: {},
+					  saveAsImage: {}
+					}
+				},
+				tooltip: {
+					trigger: 'axis',
+					animation: false,
+					axisPointer: {
+						type: 'cross',
+						label: {
+							backgroundColor: '#6a7985'
+						},
+						animation: false
+					},
+					transitionDuration: 0,
+					textStyle: {
+						fontSize: 12,
+					},
+					padding: 5,
+					order: 'seriesAsc',
+					position: [-400, '7%'],
+				},
+				axisPointer: {
+					snap: true,
+					link: [
+					  {
+						xAxisIndex: 'all'
+					  }
+					]
+				  },
+				dataZoom: [
+					{
+						type: 'inside',
+						xAxisIndex: [0, 1]
+					},
+					{
+						type: 'slider',
+						height: 20,
+						bottom: 5,
+						moveHandleSize: 4,
+						xAxisIndex: [0, 1]
+					}
+				],
+				legend: {
+					orient: 'horizontal',
+					type: 'plain',
+					x: 'center',
+					y: 'top',
+					width: '80%',
+					textStyle: {
+						color: textColor
+					},
+					selector: ['all', 'inverse']
+				},
+				/*grid: {
+					left: '20',
+					right: '10',
+					bottom: '30',
+					top: '40',
+					containLabel: true
+				},*/
+				grid: [
+					{
+					  left: 80,
+					  right: 40,
+					  height: '33%',
+					  top: '12%',
+					},
+					{
+					  left: 80,
+					  right: 40,
+					  top: '53%',
+					  height: '33%'
+					}
+				  ],
+				xAxis: [
+					{
+						type: 'category',
+						boundaryGap: false,
+						data: model.duration.domainAxisLabels,
+						axisLabel: {
+							color: textColor,
+							show: false
+						}
+					},
+					{
+						type: 'category',
+						gridIndex: 1,
+						boundaryGap: false,
+						data: model.result.domainAxisLabels,
+						axisLabel: {
+							color: textColor
+						}
+					}
+				],
+				yAxis: [
+					{
+						type: 'value',
+						min: model.duration.rangeMin ?? 'dataMin',
+						max: model.duration.rangeMax ?? 'dataMax',
+						axisLabel: {
+							color: textColor
+						},
+						minInterval: model.duration.integerRangeAxis ? 1 : null,
+						name: 'Duration (seconds)',
```

Comment:
does duration need to go to soo many decimal points?

It makes the labels extra long for limited value I expect:

<img width="1067" alt="image" src="https://github.com/jenkinsci/junit-plugin/assets/21194782/37b1c00f-783b-43ef-97ad-72279112a7d6">